docs/source/evaluations/robustness_benchmark.rst

```diff
@@ -0,0 +1,60 @@
+Benchmark: Robustness
+===========================================================
+
+VISSL provides a standardized benchmark suite to evaluate the feature representation quality of self-supervised pretrained models.
+
+One particularly important type of benchmark is robustness benchmarks.
+Robustness benchmarks tell us how well a train model generalises on samples outside of its training distribution.
+
+These benchmarks are also interesting on models pre-trained using Self-Supervised Learning and have been used in the past in papers such as `CLIP <https://cdn.openai.com/papers/Learning_Transferable_Visual_Models_From_Natural_Language_Supervision.pdf>`_:
+they give a measure of how well an SSL algorithm is able to produce representations free of spurious correlations.
+
+VISSL provides benchmarks for Out-Of-Distribution (OOD) based on the two datasets `Imagenet-A <https://github.com/hendrycks/natural-adv-examples>`_ and `Imagenet-R <https://github.com/hendrycks/imagenet-r>`_.
+In those benchmarks, a pre-trained model is either fine-tuned or trained with linear evaluation on Imagenet, and then tested against the `Imagenet-A <https://github.com/hendrycks/natural-adv-examples>`_ and `Imagenet-R <https://github.com/hendrycks/imagenet-r>`_.
+
+For reproducibility, see `VISSL Model Zoo <https://github.com/facebookresearch/vissl/blob/master/MODEL_ZOO.md>`_.
```

Comment:
is this relevant?

Reply:
You mean, the part about reproducibility?

Reply:
yep, I was wondering if we need to reference to model_zoo here :) sorry for being vague earlier :)